docs/docusaurus.config.js

```diff
@@ -134,7 +134,7 @@ const config = {
         entryPoints: ["../yarn-project/circuit-types/src/interfaces/pxe.ts"],
         tsconfig: "../yarn-project/circuit-types/tsconfig.json",
         entryPointStrategy: "expand",
-        out: "reference/developer_references/aztecjs/pxe",
+        out: "developers/reference/aztecjs/pxe",
```

Comment:
I don't see the aztec.js reference in the sidebar and going to this (and all of the aztecjs reference ones) URL gives a 404

![image](https://github.com/user-attachments/assets/4e233ea1-d24b-4ecc-896f-442dd8a029d9)